### PR TITLE
comms: All about Carbon additions, edits, deletions

### PR DESCRIPTION
**Pull request by janchild**
This PR updates old content in the All about Carbon pages. The content is repurposed from the Carbon Standard.

- What is Carbon? _Updated_
- Who uses Carbon? _New_. Content written from the perspective of each member of the product team, highlighting benefits of using Carbon, and giving them a list of ways they can engage with Carbon. 
- The Carbon ecosystem. _New_. This page repurposes the content in the Community assets section, preparing for the eventual removal of the Community assets page once the component index and the community patterns are incorporated into a new structure. There will be some duplication for now and that's fine. This is an interim step. 
- Links to the new `Who uses Carbon?` page were added to the Get started pages for both Developers and Designers. 
- And I've removed myself from the team page as I'll be heading over to .com starting June 1. 👋

### CODE REVIEW

**Review by jeanservaas — APPROVED** [review]
YAY!!!! Going out with a bang!

File: src/pages/all-about-carbon/who-uses-carbon.mdx [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+- Find [meetups](/whats-happening/meetups/) for leveling up your skills,
+  connecting with your peers, and getting reviews on work in progress.
+
+- If you're confused or if you've spotted an inconsistency, reach out on our IBM
+  Slack channels. At this time, the Slack channels are for IBMers only.
+
+  - [#carbon-design-system](https://ibm-studios.slack.com/messages/C0M053VPT/)
+  - [#carbon-components](https://ibm-studios.slack.com/archives/C046Y0YUD)
+
+- Make a [Github issue](/contributing/bugs-and-requests/) when you find a bug or
```
Comment (emyarod) [inline src/pages/all-about-carbon/who-uses-carbon.mdx:92]:
```suggestion
- Make a [GitHub issue](/contributing/bugs-and-requests/) when you find a bug or
```

File: src/pages/all-about-carbon/who-uses-carbon.mdx [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+
+  - [#carbon-design-system](https://ibm-studios.slack.com/messages/C0M053VPT/)
+  - [#carbon-components](https://ibm-studios.slack.com/archives/C046Y0YUD)
+  - [#carbon-react](https://ibm-studios.slack.com/archives/C2K6RFJ1G)
+  - [#carbon-ng](https://ibm-studios.slack.com/archives/CBZC0LM2N)
+  - [#carbon-vue](https://ibm-studios.slack.com/archives/CAM5P6NR1)
+  - [#carbon-web-components](https://ibm-studios.slack.com/archives/CL83LMKSA)
+  - [#carbon-tutorial](https://ibm-studios.slack.com/archives/CJUGA7P6H)
+
+- Make a [Github issue](/contributing/bugs-and-requests/) when you find a bug or
```
Comment (emyarod) [inline src/pages/all-about-carbon/who-uses-carbon.mdx:175]:
```suggestion
- Make a [GitHub issue](/contributing/bugs-and-requests/) when you find a bug or
```

File: src/pages/all-about-carbon/who-uses-carbon.mdx [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+easier to tell when product experiences are in lock step.
+
+Here are some ways marketers can begin engaging with Carbon:
+
+- Visit [IBM Brand Center](https://www.ibm.com/brand) to learn more about
+  foundational brand compliance, as well as for the hero brand systems that
+  drive IBM's portfolio of products and experiences.
+
+- The majority of marketing experiences use Carbon web components. Visit
+  [Carbon for ibm.com](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/)
```
Comment (emyarod) [inline src/pages/all-about-carbon/who-uses-carbon.mdx:214]:
```suggestion
  [Carbon for IBM.com](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/)
```

should this be uppercase?

File: src/pages/all-about-carbon/who-uses-carbon.mdx [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+  to ensure that these experiences are adhering to Carbon guidance.
+
+- Familiarize yourself with
+  [ibm.com web standards](https://www.ibm.com/standards/web/) to assess the
```
Comment (emyarod) [inline src/pages/all-about-carbon/who-uses-carbon.mdx:218]:
```suggestion
  [IBM.com web standards](https://www.ibm.com/standards/web/) to assess the
```

same question as above!

File: src/pages/all-about-carbon/who-uses-carbon.mdx [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+
+Consistent experiences also result in less need for extensive training and
+onboarding resources; learning can be leveraged across multiple spaces.
+
+Here are some ways people in support roles can begin engaging with Carbon:
+
+- Stay up to date with Carbon's
+  [roadmap and releases](/whats-happening/releases/).
+
+- Make a [Github issue](/contributing/bugs-and-requests/) to report bugs or
```
Comment (emyarod) [inline src/pages/all-about-carbon/who-uses-carbon.mdx:289]:
```suggestion
- Make a [GitHub issue](/contributing/bugs-and-requests/) to report bugs or
```

File: src/pages/all-about-carbon/why-use-carbon copy.mdx [hunk 6]
```diff
@@ -0,0 +1,259 @@
+---
```
Comment (emyarod) [inline src/pages/all-about-carbon/why-use-carbon copy.mdx:1]: is the `copy` at the end of the file name intentional? also does it need to be in `nav-items.yml`?

File: src/pages/all-about-carbon/the-team.mdx [hunk 7]
```diff
@@ -37,11 +37,6 @@ design and build with them.
 
 ![Andrea Cardona headshot](/images/team/cardona_andrea.png)
 
-</Profile>
-<Profile name="Jan Child" title="Content Design and Strategy Lead">
```
Comment (jillianhowarth) [inline src/pages/all-about-carbon/the-team.mdx:41]: 😢

File: src/pages/all-about-carbon/why-use-carbon copy.mdx [hunk 8]
```diff
@@ -0,0 +1,259 @@
...
+title: Why use Carbon?
```
Comment (jillianhowarth) [inline src/pages/all-about-carbon/why-use-carbon copy.mdx:2]: The page name has `copy` at the end of it. 

I'm also not seeing this page in the nav - is that intentional?

**Review by jillianhowarth — APPROVED** [review]
Looks awesome!! Noticed one file name issue but otherwise, it's great 🎉🥂👏

File: src/pages/all-about-carbon/why-use-carbon copy.mdx (hunk 8 above)
Comment (janchild) [reply src/pages/all-about-carbon/why-use-carbon copy.mdx:2]: Oh, I think I included one page by mistake. Let me check. Thank you!

File: src/pages/all-about-carbon/why-use-carbon copy.mdx (hunk 6 above)
Comment (janchild) [reply src/pages/all-about-carbon/why-use-carbon copy.mdx:1]: No, it's not intentional. Let me fix. Thanks, Andrew!

**Review by emyarod — APPROVED** [review]
looks good to me!

**Review by alisonjoseph — APPROVED** [review]

**Pull request merged**